pyvrp/cpp/educate/LocalSearch_bindings.cpp

```diff
@@ -9,10 +9,14 @@ namespace py = pybind11;
 PYBIND11_MODULE(LocalSearch, m)
 {
     py::class_<LocalSearch>(m, "LocalSearch")
-        .def(py::init<ProblemData &, PenaltyManager &, XorShift128 &>(),
+        .def(py::init<ProblemData &,
+                      PenaltyManager &, 
+                      XorShift128 &,
+                      std::vector<std::vector<int>>>(),
```

Comment:
Not sure, but can't you use the `LocalSearch::Neighbours` type here?

Reply:
```
allocator<int> >, std::allocator<std::vector<int, std::allocator<int> > > >’ is private within this context
   12 |         .def(py::init<ProblemData &,
      |                  ^~~~~~~~~~~~~~~~~~~
   13 |                       PenaltyManager &,
      |                       ~~~~~~~~~~~~~~~~~
   14 |                       XorShift128 &,
      |                       ~~~~~~~~~~~~~~
   15 |                       LocalSearch::Neighbours>(),
      |                       ~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../pyvrp/cpp/educate/LocalSearch_bindings.cpp:1:
../pyvrp/cpp/educate/LocalSearch.h:20:11: note: declared private here
   20 |     using Neighbours = std::vector<std::vector<int>>;
      |           ^~~~~~~~~~
ninja: build stopped: subcommand failed.
```
So I don't think so? We could make it public, but I'm not sure if that helps.